tests/unit/async_db/test_typing_parse.py

```diff
@@ -92,8 +94,8 @@ def test_parse_value_datetime() -> None:
         assert str(exc_info.value) == f"Invalid date value {value}: str expected"
 
     # Datetime
-    assert parse_value("2021-12-31 23:59:59", datetime) == datetime(
-        2021, 12, 31, 23, 59, 59
```

Comment:
Should the same test be there for date32?

Reply:
date32 doesn't support milliseconds

Reply:
Oh, I see what you mean